tests/test-team/setup/session.setup.ts

```diff
@@ -0,0 +1,14 @@
+import { test as setup } from '@playwright/test';
+import { TemplateMgmtChoosePage } from '../pages/template-mgmt-choose-page';
+import { TemplateMgmtCreatePage } from '../pages/template-mgmt-create-page';
+import { Session } from '../helpers/types';
+import SessionStorageHelper from '../helpers/session-storage-helper';
+
+setup('setup session data', async () => {
+  const sessionData: Session[] = [
+    ...TemplateMgmtChoosePage.sessionData,
```

Comment:
This seems a little strange having session data on the `TemplateMgmtChoosePage` POM. 

Is it worth splitting out the session data into a session fixture? 

I'll add the suggestion to your ticket.

Reply:
updated - test suites now seed their own data, none of this is global data so we don't really need a global setup at the moment